### ISSUE TITLE
Fourier transform example?

### ISSUE THREAD

**Issue by Xeoncross** [mjibson/go-dsp#19]
I would love an example of using the Fourier transform utilities on one of the sample wav files. In fact, there are several algorithms here that could use a usage demonstration.

**Comment by mjibson** [comment]
Yes, these would be nice. However I won't be working on these soon. I'd review a PR though.
